tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_jinja_filters.py

```diff
@@ -111,4 +111,30 @@ def translate_column_bool(column_name):
     """
 
 
+SQL_LINK_FORMAT = """
+concat('<a href="{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ SUPERSET_HOST }}/superset/dashboard/{dashboard_slug}/?native_filters=(NATIVE_FILTER-{filter_id}:(__cache:(label:''', 
+    {column}, 
+    ''',validateStatus:!f,value:!(''', 
+    {column}, 
+    ''')),extraFormData:(filters:!((col:{column},op:IN,val:!(''', 
+    {column}, 
+    ''')))),filterState:(label:''', 
+    {column}, 
+    ''',validateStatus:!f,value:!(''', 
+    {column}, 
+    ''')),id:NATIVE_FILTER-IfS-Rd0ZS,ownState:()))">', 
+    {column}, 
+    '</a>'
+)
+"""
+def filtered_dashboard(dashboard_slug, column_name, filter_id):
```

Comment:
Can you rename this to something more descriptive like "get_filtered_dashboard_link"?